scout/server/blueprints/cases/templates/cases/case_sma.html

```diff
@@ -2,7 +2,6 @@
 {% from "cases/collapsible_actionbar.html" import action_bar, research_modal, rerun_modal %}
 {% from "utils.html" import comments_panel, activity_panel %}
 {% from "cases/utils.html" import individuals_table %}
-{% from "cases/case_ideograms.html" import case_ideograms %}
```

Comment:
An ancient relic from the past?

Reply:
Yes, it did not seem to be doing much, and there was a clear phylogenetic similarity to the case_images.js that is actually in use. I actually made the edits in that one first.. 🙄